src/sentry/search/events/builder.py

```diff
@@ -109,6 +109,7 @@ def __init__(
         turbo: bool = False,
         sample_rate: Optional[float] = None,
         equation_config: Optional[Dict[str, bool]] = None,
+        skip_time_conditions: bool = False,
```

Comment:
Could we rename the variable, or add a comment explaining that this is specifically for alerts?